client/src/App.js

```diff
@@ -1,7 +1,76 @@
 import React from 'react';
+import styled from '@emotion/styled';
+import Particles from 'react-particles-js';
+import { useState, useEffect } from 'react';
+import colors from './utils/colorsUtils';
+
+function getWindowDimensions() {
+  const { innerWidth: width, innerHeight: height } = window;
+  return {
+    width,
+    height,
+  };
+}
+
+function useWindowDimensions() {
+  const [windowDimensions, setWindowDimensions] = useState(
+    getWindowDimensions()
+  );
+
+  useEffect(() => {
+    function handleResize() {
+      setWindowDimensions(getWindowDimensions());
+    }
+
+    window.addEventListener('resize', handleResize);
+    return () => window.removeEventListener('resize', handleResize);
+  }, []);
+
+  return windowDimensions;
+}
+
+const Test = styled.div`
+  width: 100px;
+  height: 100px;
+  background-color: red;
+`;
 
 function App() {
-  return <div>Hello</div>;
+  const { height, width } = useWindowDimensions();
+  const particleNumbers = (height + width) / 15;
+  console.log(particleNumbers);
+  return (
+    <>
+      <Test />
+      <h1 style={{ color: 'white', margin: '50px' }}>Hello</h1>
+      <Particles
+        style={{
+          background: `${colors.primaryDark}`,
+          zIndex: '-10',
```

Comment:
zIndex of -1 would be enough.

Reply:
better safe than sorry.
Who knows, what animation thing comes next in my mind...

Will change it to -1 though 😉